app/controllers/occupations_controller.rb

```diff
@@ -0,0 +1,9 @@
+class OccupationsController < ApplicationController
+  def index
+    es_response = OccupationElasticsearchQuery.new(
+      search_params: params.permit(:q).to_h
+    ).call
+    occupations = es_response.records.to_a
```

Comment:
*nit*: Ypu may not need to call `to_a`. I removed it for occupation standards index and is working 😄

Reply:
The tests were failing when I tried to to remove `to_a` from occupations